_posts/2021-07-02-status-update.md

```diff
@@ -0,0 +1,104 @@
+author: "binford2k"
+categories:
+  - team
+  - status
+tags:
+  - devx
+  - community
+  - dsc
+  - k8s
+  - cloudci
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-docker#740`][puppetlabs-docker-pr-740]: "Allow management of the docker-ce-cli package", thanks to [kenyon][kenyon]
+- [`puppetlabs-iis#325`][puppetlabs-iis-pr-325]: "Correct error handling in require ruby-pwsh", thanks to [benningm][benningm]
+- [`puppetlabs-inifile#454`][puppetlabs-inifile-pr-454]: "Accept Datatype Sensitive", thanks to [cocker-cc][cocker-cc]
+- [`puppetlabs-java_ks#373`][puppetlabs-java_ks-pr-373]: "Accept Datatype Sensitive for Secrets", thanks to [cocker-cc][cocker-cc]
+- [`puppetlabs-java_ks#372`][puppetlabs-java_ks-pr-372]: "Fix fingerprint comparison", thanks to [kdehairy][kdehairy] and the following people who helped get it over the line ([creativefre][creativefre])
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-docker`][puppetlabs-docker] (`4.1.0`)
+- [`puppetlabs-inifile`][puppetlabs-inifile] (`5.1.0`)
+- [`puppetlabs-java_ks`][puppetlabs-java_ks] (`4.1.0`)
+- [`puppetlabs-iis`][puppetlabs-iis] (`8.0.3`)
+- [`puppetlabs-helm`][puppetlabs-helm] (`4.0.0`)
+
+  [puppetlabs-docker]: https://github.com/puppetlabs/puppetlabs-docker
+  [puppetlabs-inifile]: https://github.com/puppetlabs/puppetlabs-inifile
+  [puppetlabs-java_ks]: https://github.com/puppetlabs/puppetlabs-java_ks
+  [puppetlabs-iis]: https://github.com/puppetlabs/puppetlabs-iis
+  [puppetlabs-helm]: https://github.com/puppetlabs/puppetlabs-helm
+  [puppetlabs-docker-pr-740]: https://github.com/puppetlabs/puppetlabs-docker/pull/740
+  [kenyon]: https://github.com/kenyon
+  [puppetlabs-iis-pr-325]: https://github.com/puppetlabs/puppetlabs-iis/pull/325
+  [benningm]: https://github.com/benningm
+  [puppetlabs-inifile-pr-454]: https://github.com/puppetlabs/puppetlabs-inifile/pull/454
+  [cocker-cc]: https://github.com/cocker-cc
+  [puppetlabs-java_ks-pr-373]: https://github.com/puppetlabs/puppetlabs-java_ks/pull/373
+  [puppetlabs-java_ks-pr-372]: https://github.com/puppetlabs/puppetlabs-java_ks/pull/372
+  [kdehairy]: https://github.com/kdehairy
+  [creativefre]: https://github.com/creativefre
+
+## Community Day
+
+We had a great community day this week and processed 28 issues and PRs across our supported modules and tooling.
+And a shout out again to [cocker-cc][cocker-cc] for their continued engagement in adding `Sensitive` Datatype support to more modules.
+
+
+## Kubernetes Conformance
+
+We're very happy to announce that we finished running the conformance tests for v1.20 on our kubernetes module and we're just one step a way of the official news.
```

Comment:
It's official ! We're very happy to announce that we finished running the conformance tests for v1.20 on our kubernetes module and we became certified this morning.

Reply:
Not sure how to make it sound better

Reply:
Its official Puppet has been re-awarded CNCF certification for our supported Kubernetes module v1.20.